src/main/java/subway/path/ui/PathController.java

```diff
@@ -30,7 +31,8 @@ public PathController(PathService pathService) {
                                                           .stream()
                                                           .map(StationResponse::of)
                                                           .collect(Collectors.toList());
-    int fare = shortestPath.getFare(loginMember.getAge());
+    AgeGroup ageGroup = loginMember == null ? AgeGroup.ADULT : AgeGroup.getAgeGroup(loginMember.getAge());
```

Comment:
ternary operator( `? :` )는 사실 if~else와 다를바 없긴 하지만..
```suggestion
    AgeGroup ageGroup = AgeGroup.ADULT;
    if(loginMember != null) {
        ageGroup = AgeGroup.getAgeGroup(loginMember.getAge());
    }
```
우리 모두 if문 만으로도 충분히 대체 가능하단 사실을 알고 있기 때문에 별 피드백은 하지 않겠습니다 :)